user_docs/en/userGuide.t2t

```diff
@@ -1074,6 +1074,27 @@ When you scroll the display, you will then be able to read the rest of the word.
 
 Enabling this may allow for more fluent reading, but generally requires you to scroll the display more.
 
+==== Focus context presentation ====
+This option allows you to choose what context information NVDA will show on the braille display when an object gets focus.
+Context information (or focus ancestry) refers to the hierarchical tree of objects containing the focus object.
+For example, when you just focused a list item, this list item is part of a list. This list might be contained by a dialog, etc.
+Please consult the section about [object navigation #ObjectNavigation] for more information about the hierarchy that applies to objects in NVDA.
+
+When set to fill display for context changes, NVDA will try to display as much as possible context information on the braille display, but only for the parts of the context that have changed.
+For the example above, this means that when changing focus to the list, NVDA will show the list item on the braille display.
+Furthermore, if there is enough space left on the braille display, NVDA will try to show that the list item is part of a list.
+Now, if you start moving through the list with your arrow keys, it is assumed that you are aware of the fact that you are moving through a list.
+Thus, for the remaining list items you focus, NVDA will only show the focused list item on the display.
+In order for you to read the context again (i.e. that you are in a list and that the list is part of a dialog), you will have to scroll your braille display back.
+
+When this option is set to always fill the display, NVDA will try to show as much as possible context information on the braille display, regardless of whether you have seen the same context information before.
+This has the advantage that NVDA will make use of the braille display in an effective way, filling up as much as possible empty display space.
+however, this option has the downside that there is always a difference in the position where the representation of the object with focus starts on the braille display.

```

Comment:
1. Need capital at start of sentence.
2. I think we can simplify/shorten this:

> `However, the downside is that there is always a difference in the position where the focus starts on the braille display.`

Would it also be useful to mention that this makes skimming difficult? For example:

> `This can make it difficult to skim a long list of items, for example, as you will need to continually move your finger to find the start of the item.`